universal-profiling-integration/src/test/java/co/elastic/otel/UniversalProfilingProcessorTest.java

```diff
@@ -239,37 +247,36 @@ private String readUtf8Str(ByteBuffer buffer) {
       buffer.get(serviceUtf8);
       return new String(serviceUtf8, StandardCharsets.UTF_8);
     }
+  }
 
-    private void checkTlsIs(Span span, Span localRoot) {
-      ByteBuffer tls =
-          JvmtiAccessImpl.createThreadProfilingCorrelationBufferAlias(TLS_STORAGE_SIZE);
-      if (tls != null) {
-        tls.order(ByteOrder.nativeOrder());
-        assertThat(tls.getChar(0)).isEqualTo((char) 1); // layout-minor-version
-        assertThat(tls.get(2)).isEqualTo((byte) 1); // valid byte
-      }
+  static void checkTlsIs(Span span, Span localRoot) {
+    ByteBuffer tls = JvmtiAccessImpl.createThreadProfilingCorrelationBufferAlias(TLS_STORAGE_SIZE);
+    if (tls != null) {
+      tls.order(ByteOrder.nativeOrder());
+      assertThat(tls.getChar(0)).isEqualTo((char) 1); // layout-minor-version
```

Comment:
[nit] use `.describedAs("layout-minor-version")` to describe the assertion instead of comment.